aws-blog-mirth-healthcare-hub/bb-webapp/bb-conversion-listener.js

```diff
@@ -0,0 +1,49 @@
+
+//Lets require/import the HTTP module
+var http = require('http');
+var qs = require('querystring');
+var bb = require('blue-button');
+
+//Lets define a port we want to listen to
+const PORT=8000; 
+var conn_count = 0;
+//We need a function which handles requests and send response
+function handleRequest(request, response){
```

Comment:
Please format this file according to java script code style.

http://standardjs.com/